client/commonFramework/bindings.js

```diff
@@ -150,6 +150,10 @@ window.common = (function(global) {
       window.ga('send', 'event', 'Challenge', 'load', common.gaName);
     }
 
+    $('.modal').on('show.bs.modal', function() {
+      $('aside').addClass('is-collapsed');
```

Comment:
@BerkeleyTrue 

Do you mean the 'aside' selector? I can definitely see that being too generic and can change that.

How about the '.modal' selector? I didn't make it specific because I figured that if any modal is open, you want all those asides to collapse.